src/LaTeXStrings.jl

```diff
@@ -96,7 +96,13 @@ Base.show(io::IO, ::MIME"application/x-latex", s::LaTeXString) = print(io, s.s)
 Base.show(io::IO, ::MIME"text/latex", s::LaTeXString) = print(io, s.s)
 function Base.show(io::IO, s::LaTeXString)
     print(io, "L")
-    Base.print_quoted_literal(io, s.s)
+    print_quoted_literal(io, s.s)
```

Comment:
I think the right thing is probably `Base.escape_raw_string`, which was introduced in Julia 1.4.  That is, we can do:
```jl
if isdefined(Base, :escape_raw_string)
    import Base: escape_raw_string
else
    const escape_raw_string = Base.print_quoted_literal
end
```

Reply:
Still had to do some shenanigans, because `escape_raw_string` doesn't print the leading and trailing `'"'`. This is slightly more code, but I agree it's probably good to still use the `Base` escaping functions.

Reply:
In that case, it's probably cleaner to just put this all in the `show` function.   i.e.
```jl
function Base.show(io::IO, s::LaTeXString)
    @static if isdefined(Base, :escape_raw_string)   # Julia ≥ 1.4
        print(io, '"')
        Base.escape_raw_string(io, s)
        print(io, '"')
    else
        Base.print_quoted_literal(io, s) # removed in Julia 1.6
    end
end
```

Reply:
Sounds good to me.